docs/webapp/webapp_profile.md

```diff
@@ -238,6 +240,90 @@ Removed users lose access to your workspace's resources (tasks, models, etc.) an
 revoked. Tasks and associated artifacts logged to your workspace by a removed user will remain in your workspace. The 
 user can only rejoin your workspace when you re-invite them. 
 
+### Service Accounts
+
+:::important Enterprise Feature
+This feature is available under the ClearML Enterprise plan.
+:::
+
+Service accounts are ClearML users that provide services with access to the ClearML API, but not the 
+UI. Administrators can create access credentials for service accounts to use them for different ClearML Agents, 
+automations, and more. 
+
+A service account has all the privileges of a normal user in ClearML, with the following limitations:
+* Service accounts cannot be used to access the UI
+* Service accounts can be used to facilitate running tasks under the identity of each task's owner ("Impersonation"). 
+  This has the added benefit of allowing you to specify Vaults that apply to this service account, thus managing 
```

Comment:
@pollfly Impersonation facilitates applying the owner's vault rather than the accounts vault, no?

Reply:
@ainoam I've completely removed the sentence about the vaults. The information is covered under the "Impersonation" admonition `When impersonation is enabled... it will have access to the task owner's configuration vaults`